CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.4] - 2024-01-31
+
```

Comment:
Please add bug numbers/URLs to the three bug fixes if there are bug reports.

Reply:
I can't find the bug report for the bug fix at line 13.

https://github.com/AOMediaCodec/libavif/pull/1956 says the fix at line 15 is for b/321189607 and b/321189607.

The fix at line 16 is for https://crbug.com/oss-fuzz/65657.

Reply:
b/320234262 is for https://github.com/AOMediaCodec/libavif/pull/1942

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1983 and https://github.com/AOMediaCodec/libavif/pull/1984

Reply:
Rebased